nifi-framework-bundle/nifi-framework/nifi-web/nifi-jetty/src/main/java/org/apache/nifi/web/server/StandardServerProvider.java

```diff
@@ -63,7 +63,7 @@ class StandardServerProvider implements ServerProvider {
     public Server getServer(final NiFiProperties properties) {
         Objects.requireNonNull(properties, "Properties required");
 
-        final QueuedThreadPool threadPool = new QueuedThreadPool(properties.getWebThreads());
+        final VirtualThreadPool threadPool = new VirtualThreadPool(properties.getWebThreads());
```

Comment:
I recommend leaving this as the `QueuedThreadPool` for now, because the web threads property is less meaningful with Virtual Threads. It may be worth considering deprecating it. Either way, it seems like something better to consider in a separate issue.